include/rocksdb/utilities/stackable_db.h

```diff
@@ -383,6 +383,11 @@ class StackableDB : public DB {
     return db_->GetCurrentWalFile(current_log_file);
   }
 
+  virtual Status GetFilesViolatingTtl(
+      std::vector<std::string>& files) override {
```

Comment:
I feel that the API is a little bit too strong to me. "Violating" is a strong word, which only users can determine. 

Do we consider an alternative solution, that is to return the creation time of the oldest file? We probably don't need full list of files.